lib/routing/services/routing.dart

```diff
@@ -479,7 +479,7 @@ class Routing with ChangeNotifier {
 
           // Only create instructions if the user has enabled audio.
           List<Instruction> instructions = List<Instruction>.empty(growable: true);
-          if (getIt<Settings>().audioInstructionsEnabled) {
+          if (getIt<Settings>().audioRoutingInstructionsEnabled) {
```

Comment:
I think it would make sense to remove the routing audio instructions code in favor of a cleaner code base. In the project we won't need it any more I think and even if, we can still get the code from past commits. What do you think? @PaulPickhardt